projects/sturdy/index.js

```diff
@@ -24,27 +27,37 @@ const priceInUSD = (value, decimals, price) => {
     return (value / Math.pow(10, decimals) * price / Math.pow(10, 8)).toFixed(2);
 }
 
-async function fetch(borrow) {
+async function fetch(borrow, chain) {
+    const graphUrl = chain == 'fantom' ? graphUrl_ftm : graphUrl_eth;
     const { reserves } = await request(graphUrl, graphQuery);
-    const tvl = reserves.reduce((sum, reserve) => {
+    let tvl = reserves.reduce((sum, reserve) => {
         const value = borrow ? reserve.availableLiquidity : reserve.totalLiquidity;
```

Comment:
Also, I think the value here should be `reserve.totalLiquidity - reserve.availableLiquidity` for `borrowed` and `reserve.availableLiquidity` for tvl?

tvl should not include the amount that has been borrowed